.eslintrc

```diff
@@ -14,10 +14,13 @@
       "ignore": ["url"] }
     ],
     "no-return-assign": "off",
-    "radix": ["error", "as-needed"]
-  },
+    "radix": ["error", "as-needed"],
+    "header/header": [2, "line", [{"pattern": " Copyright 2020 The OpenZipkin Authors; licensed to You under the Apache License, Version 2.0."}]]},
```

Comment:
how do we want to handle this date.. manually change it? In other projects we are deriving this from git commit info, so that it says like 2016-2020 on file change.

(feel free to merge without addressing this question)

Reply:
> In other projects we are deriving this from git commit info. 

That‘s great.  and I'm a little confused. Do we really have to keep copyright year and then change it once a year? Personally I suggest we remove copyright Year like other projects. [open-telemetry/opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java) : )

Reply:
I suspect there's something stronger by saying the year range. However, I'd suggest either do it right or leave it out. I wouldn't necessarily do anything just because otel does though :D

Reply:
Thanks for sharing your thought. This is only a suggestion, for reference only. : ). 
> In other projects we are deriving this from git commit info. 

We can try to do this. I know maven plugin can do this (not sure js). It would be better If you can give me a link?

Reply:
cool. here's the code used in the maven plugin, if that helps.. https://github.com/mycila/license-maven-plugin/blob/e4675b61308ccd6bbf0fd2bd6a1aa309973ea048/license-maven-plugin-git/src/main/java/com/mycila/maven/plugin/license/git/CopyrightRangeProvider.java#L79-L80

Reply:
note if we do this, we need a well commented thing on jobs to do full, not shallow git clone. This is a pitfall of using the git range, as it can be inaccurate if for example, you only get the last 5 commits.